Modules/DataModel/Sources/PocketCastsDataModel/Public/PlaylistQueryBuilder.swift

```diff
@@ -35,7 +35,8 @@ public class PlaylistQueryBuilder {
         let select = select(clause: clause)
         let addedUuid = add(episodeUuidToAdd: episodeUuidToAdd)
         let rules = add(smartRulesFor: playlist)
```

Comment:
@bjtitus are we sure that this builder is ok to be used for manual playlists? For example you didn't filter the smart rules method, which is not necessary for manual playlist. Also there's an extra `AND` when filtering out the rules when fetching a manual playlist like (which generates an error):
```
SELECT episode.* FROM SJEpisode episode LEFT JOIN SJPodcast podcast ON episode.podcast_id = podcast.id WHERE episode.archived = 0 AND ( ) AND episode.uuid IN ('e92dfec3-0b3f-412c-b63d-236fe8d4318c') ORDER BY episode.publishedDate DESC, episode.addedDate DESC  LIMIT 1000
```
Compared to the Android query for manual, down need to do something else or is this enough?